<!-- ssd_node.launch -->
<launch>

    <group ns = "ssd_object_detect">

        <param name="ssd_img_show_flag" type="bool" value="false"/>
        <param name="ssd_execute_default" type="bool" value="true"/>
        <param name="ssd_pub_result_image" type="bool" value="true"/>
        <param name="needs_time_stamp" type="bool" value="false"/>

        <param name="ssd_image_topic_name" type="str" value="/camera/rgb/image_raw"/>
        <param name="ssd_inScaleFactor" type="double" value="0.007843"/>
        <param name="ssd_confidenceThreshold" type="double" value="0.5"/>

        <param name="ssd_prototxt_name" type="str" value="voc_object.prototxt"/>
        <param name="ssd_caffemodel_name" type="str" value="voc_object.caffemodel"/>
        <param name="ssd_classNames_file" type="str" value="voc_object_names.txt"/>

        <node pkg="ssd_node" type="ssd_node" name="ssd_node" output="screen"/>
    </group>
</launch>
